### ISSUE TITLE
8xl files can't be sent :(

### ISSUE THREAD

**Issue by Mazedotexe** [Timendus/ticalc.link#65]
I used to use TI Connect CE, but when I found this, I started only using it because it was much easier and it was in HTML5, which meant I could use it on my school computer. Eventually my PC broke, and I was forced to use this site. It's wonderful, but surprising to me that after all this time TI 84 Plus CE is still in beta; I would imagine it's what the majority of ticalc.link users use it for. Over the past 4 months or so I've been really interested in Brainfuck and how it's so complicated to do simple things like the Fibonacci sequence. So I tried to install [Brainfoo](https://www.cemetech.net/downloads/files/2286/x3008) for the CE. But it has necessary 8xl files, which unfortunately cannot be sent on ticalc.link. I started work on my own brainfuck interpreter, but eventually dropped it, being much harder than I thought. I've came back to it a few times, but I doubt I'll ever finish it with the knowledge that a better one already exists, but I just can't install it. Eventually I decided to come here and make this issue to ask if support for CE files like .8xl will come any time soon. I hope I get a response :) I've checked and made sure this isn't a copy of another. At least, it's not mentioned in [this issue.](https://github.com/Timendus/ticalc.link/issues/38)

TL;DR, I'd love to know when (or if) support for more obscure CE file extensions will come, as they're sometimes (albeit rarely) necessary.
